packages/scaffold-ui/src/views/w3m-profile-view/index.ts

```diff
@@ -0,0 +1,165 @@
+import {
+  AccountController,
+  ConnectorController,
+  CoreHelperUtil,
+  ModalController,
+  RouterController,
+  SnackController,
+  type AccountType
+} from '@web3modal/core'
+
+import { UiHelperUtil, customElement } from '@web3modal/ui'
+import { LitElement, html } from 'lit'
+import { state } from 'lit/decorators.js'
+import { ifDefined } from 'lit/directives/if-defined.js'
+import styles from './styles.js'
+
+@customElement('w3m-profile-view')
+export class W3mProfileView extends LitElement {
+  public static override styles = styles
+
+  // -- Members -------------------------------------------- //
+  private usubscribe: (() => void)[] = []
+
+  // -- State & Properties --------------------------------- //
+  @state() private address = AccountController.state.address
+
+  @state() private profileImage = AccountController.state.profileImage
+
+  @state() private profileName = AccountController.state.profileName
+
+  @state() private accounts = AccountController.state.allAccounts
+
+  public constructor() {
+    super()
+    this.usubscribe.push(
+      AccountController.subscribe(val => {
+        if (val.address) {
+          this.address = val.address
+          this.profileImage = val.profileImage
+          this.profileName = val.profileName
+        } else {
+          ModalController.close()
+        }
+      })
```

Comment:
Can we use `subscribeKey` to not listen unnecessary changes of the AccountController please?
```suggestion
      AccountController.subscribeKey("address", val => {...}),
      AccountController.subscribeKey("profileImage", val => {...}),
      AccountController.subscribeKey("profileName", val => {...}),
```